### PR TITLE
Fix end positions for `at-rule-*-list`

### PR DESCRIPTION
**Pull request by mattxwang**
> Which issue, if any, is this issue related to?

Part of the umbrella issue #5694.

> Is there anything in the PR that needs further explanation?

I modelled my approach off of #6026 - though, I did note that for the rule in that PR (`at-rule-no-unknown`), the `message` includes the `@`, while for the rules in this PR, the `message` does not. I've still included the `@` in the word itself; let me know if I should instead be consistent with the `message`.

(also not sure about the add/fix terminology for the PR title, feel free to change if needed)

### CODE REVIEW

File: lib/rules/at-rule-allowed-list/index.js [hunk 1]
```diff
@@ -47,6 +47,7 @@ const rule = (primary) => {
 				node: atRule,
 				result,
 				ruleName,
+				word: `@${atRule.name}`,
```
Comment (ybiquitous) [inline lib/rules/at-rule-allowed-list/index.js:50]: [trivial] Reusing `name` seems good 👍🏼 

```suggestion
				word: `@${name}`,
```

**Review by ybiquitous — COMMENTED** [review]
@mattxwang Thanks, LGTM 👍🏼 

IMO, it's easier to understand if each rule message includes `@`, e.g.

```diff
- Unexpected at-rule "foo"
+ Unexpected at-rule "@foo"
```

File: lib/rules/at-rule-disallowed-list/index.js [hunk 2]
```diff
@@ -47,6 +47,7 @@ const rule = (primary) => {
 				node: atRule,
 				result,
 				ruleName,
+				word: `@${atRule.name}`,
```
Comment (ybiquitous) [inline lib/rules/at-rule-disallowed-list/index.js:50]: Ah, this also can be simplified: 😅 

```suggestion
				word: `@${name}`,
```

**Review by ybiquitous — APPROVED** [review]

**Review by jeddy3 — APPROVED** [review]
LGTM, thank you!

**Comment by jeddy3** [comment]
> - Fixed: `at-rule-*-list` end positions ([#6032](https://github.com/stylelint/stylelint/pull/6032)).

**Pull request merged**